front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -309,22 +310,48 @@ export default function LabsTranscriptsIndex({
       if (transcriptsConfigurationState.provider !== "gong") {
         return;
       }
-      const nango = new Nango({ publicKey: nangoPublicKey });
-      const newConnectionId = buildLabsConnectionId(
-        `labs-transcripts-workspace-${owner.id}-user-${user.id}`,
-        transcriptsConfigurationState.provider
-      );
-      const {
-        connectionId: nangoConnectionId,
-      }: { providerConfigKey: string; connectionId: string } = await nango.auth(
-        nangoGongConnectorId,
-        newConnectionId
-      );
 
-      await saveOauthConnection(
-        nangoConnectionId,
-        transcriptsConfigurationState.provider
+      const response = await fetch(
+        `/api/w/${owner.sId}/labs/transcripts/default`
       );
+
+      if (response.ok) {
+        const defaultConfigurationRes = await response.json();
+        const defaultConfiguration: LabsTranscriptsConfigurationResource =
+          defaultConfigurationRes.configuration;
+
+        if (defaultConfiguration.provider !== "gong") {
```

Comment:
So you can have only one transcript per workspace?

Reply:
Yes for now